frontend/src/charts/ChoroplethMap.tsx

```diff
@@ -14,6 +15,7 @@ const MISSING_DATASET = "MISSING_DATASET";
 const VALID_DATASET = "VALID_DATASET";
 const GEO_DATASET = "GEO_DATASET";
 const GEO_ID = "id";
+const COLOR_SCALE = "COLOR_SCALE";
```

Comment:
Why did this change from "colorScale" to "COLOR_SCALE"? Does this work by referencing the below variable by the "name" property?

Reply:
Yeah I was trying to change the arbitrary string names to the variables to reduce human error.